var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -572,6 +572,8 @@ def use_x11():
 
         if "+adios2" in spec:
             cmake_args.extend(["-DPARAVIEW_ENABLE_ADIOS2:BOOL=ON"])
+            if spec.variants["use_vtkm"].value != "off" and spec.satisfies("@5.9:"):
+                cmake_args.append("-DPARAVIEW_ENABLE_FIDES:BOOL=ON")
```

Comment:
```suggestion
        if spec.satisfies("@5.9:") and spec.variants["use_vtkm"].value != "default":
            cmake_args.append(
                "-DPARAVIEW_ENABLE_FIDES:BOOL=%s" % spec.variants["use_vtkm"].value.upper())
```
This might be more accurate.

Reply:
when use_vtkm is default there is no guarantee of being enabled.

Reply:
Hmm. Good point. The intention was to build VTK-m if Fides is requested. That seems to be the case for the `+cuda` and `+rocm` options. How do those things work?

Reply:
We disallow configuring GPU with vtk-m off.

```python
conflicts("+cuda", when="use_vtkm=off")
conflicts("+rocm", when="use_vtkm=off")
```

Then the default when using GPU enabled filters is to use VTKm filters. So the default is on in that case.

It is probably worth conflicting `default` in that case, I think it is strange that `+cuda use_vtkm=default` and `+cuda use_vtkm=on` are two distinct specs, but the same build.

Reply:
OK. I see my approach isn't working.

I redesigned to have a `+fides` variant that will check to make sure that adios2 and vtk-m are available.